### ISSUE TITLE
Highlight histogram range based on percentiles

### ISSUE THREAD

**Issue by jtbandes** [getguesstimate/guesstimate-app#102]
It would be nice to see the actual range (e.g. ±13) displayed on the histogram, either for 90% or for the particular percentile that I'm looking at (hover state on the percentile table).

![image](https://cloud.githubusercontent.com/assets/14237/13312988/9995e5ec-db4f-11e5-8b5d-25de838587b6.png)

**Comment by OAGr** [comment]
Agreed this could use quite a bit of work.  Would be sweet to use D3 or something similar to have a hover effect so you can see it with more precision.
